conan/cli/commands/test.py

```diff
@@ -16,7 +16,8 @@ def test(conan_api, parser, *args):
     Test a package from a test_package folder.
     """
     parser.add_argument("path", action=OnceArgument,
-                        help="Path to a test_package folder containing a conanfile.py")
+                        help="Path to a test_package folder containing a conanfile.py",
```

Comment:
This might be confusing, because ``conan test`` won't work without changing to the ``test_package`` folder, the most common usage is ``conan test test_package``